src/storage/disk_table.h

```diff
@@ -409,27 +409,33 @@ class DiskTable : public Table {
     bool IsExpire(const ::openmldb::api::LogEntry& entry) override;
 
     void CompactDB() {
-        for (rocksdb::ColumnFamilyHandle* cf : cf_hs_) {
+        for (rocksdb::ColumnFamilyHandle* cf : *cf_hs_) {
             db_->CompactRange(rocksdb::CompactRangeOptions(), cf, nullptr, nullptr);
         }
     }
 
     int CreateCheckPoint(const std::string& checkpoint_dir);
 
+    bool AddIndex(const ::openmldb::common::ColumnKey& column_key) override;
     bool DeleteIndex(const std::string& idx_name) override;
     uint64_t GetRecordIdxCnt() override;
     bool GetRecordIdxCnt(uint32_t idx, uint64_t** stat, uint32_t* size) override;
     uint64_t GetRecordPkCnt() override;
     uint64_t GetRecordByteSize() const override { return 0; }
     uint64_t GetRecordIdxByteSize() override;
 
+    std::vector<rocksdb::ColumnFamilyHandle*> GetCF_DS_() {
```

Comment:
naming convention: `GetCfDs()`.

here it is more like `CloneCf`?

Reply:
Yes, it looks more reasonable.

Reply:
> naming convention: `GetCfDs()`.
> 
> here it is more like `CloneCf`?

My fault. Here should have be `GetCF_HS_` and actually, `GetCF_DS_` returns a copy of `cf_hs`.